xml/System.Runtime.CompilerServices/Unsafe.xml

```diff
@@ -60,12 +71,22 @@
         <Parameter Name="elementOffset" Type="System.Int32" Index="1" FrameworkAlternate="dotnet-plat-ext-2.1;dotnet-plat-ext-2.2;net-5.0;net-6.0;netcore-3.0;netcore-3.1" />
       </Parameters>
       <Docs>
-        <typeparam name="T">The type of void pointer.</typeparam>
-        <param name="source">The void pointer to add the offset to.</param>
+        <typeparam name="T">The type whose width will be used as a scale factor for `elementOffset`.</typeparam>
+        <param name="source">The unmanaged pointer to add the offset to.</param>
         <param name="elementOffset">The offset to add.</param>
-        <summary>Adds an element offset to the given void pointer.</summary>
-        <returns>A new void pointer that reflects the addition of offset to the specified pointer.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Adds an element offset to the given unmanaged pointer.</summary>
```

Comment:
if we're touching this anyways, can we order this "properly":
```
<summary></summary>
<typeparam name=""></typeparam>
<param name=""></param>
<returns></returns>
<remarks></remarks>
```

Reply:
(Noting "properly" is defined here as the order VS declares these by default when you type `///` and matches a good reading order for understanding the method)

Reply:
@gewarren Thoughts on this? I'm looking through a small sample of other .xml files in the repo, and the consistent pattern I see is "typeparam, param, summary, returns, remarks." If we wanted to reorder this to be more proper I wonder if that should be something that's done repo-wide rather than as a one-off inside this file.

Reply:
Yes, I don't think it's worthwhile to reorder the tags. Also, I have a suspicion that mdoc will just reorder them the next time we update the APIs.

Reply:
Is it worth a tracking issue for mdoc to be updated? It's definitely much harder to read/review these files because of the difference

Reply:
Yes - I logged https://dev.azure.com/ceapex/Engineering/_workitems/edit/619653.